resources/energyplus/ProposedEnergy+.idd

```diff
@@ -11422,33 +11422,92 @@ People,
        \object-list ScheduleNames
        \note units in the schedule are m/s
        \note optional (only required for runs of thermal comfort models: Fanger, Pierce, KSU, CoolingEffectASH55 and AnkleDraftASH55)
-  A14, \field Ankle Level Air Velocity Schedule Name
+  A14, \field Thermal Comfort Model 1 Type
+       \type choice
+       \key Fanger
+       \key Pierce
+       \key KSU
+       \key AdaptiveASH55
+       \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
+       \note optional (only needed for people thermal comfort results reporting)
+  A15, \field Thermal Comfort Model 2 Type
+       \type choice
+       \key Fanger
+       \key Pierce
+       \key KSU
+       \key AdaptiveASH55
+       \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
+       \note optional (second type of thermal comfort model and results reporting)
```

Comment:
E+ added three new fields after these, which we originally treated (as it should have been) as `\extensible-fields`. We unfortunately cannot do that anymore, so put back the 7 individual fields. That's why most epJSON tests where failing.